### ISSUE TITLE
Receiving "Error when using  http://komputing.org:8081

### ISSUE THREAD

**Issue by poiresel** [ethereum/source-verify#3]
Issue: "Error: Metadata file not found. Did you include "metadata.json"?" when using the UI

Screen shot of inputs:
![sourceverify](https://user-images.githubusercontent.com/2044804/68956021-33df1180-0795-11ea-8430-a5ffa9155f55.png)

System : Tried on both Chrome and Firefox same issue

metadata.json was generated using `solc --metadata --metadata-literal`

as well as passed it into source-verify/index.js < metadata.json with output

```
Metadata MATCHES!
Bytecode comparison NOT performed - has to be done manually.
```

Sample contract used was 0xd2006b3452ff12ce48d018ff48e0df76bca6ee04 where we've already uploaded the source to ipfs here: https://ipfs.infura.io/ipfs/Qmbu6TMyKCH9HJVvhG6BFA3UkrJuGVdZjhFQqKRRKmsNyD

**Comment by chriseth** [comment]
Ah, thanks for trying out this beta software! The `--metadata-literal` version is not yet implemented because I thought this would be the more difficult way to use it :)
Did you also use `--metadata-literal` when you compiled the contract? Would you be so kind and attach both the metadata file that was used during compilation for deployment (this is crucial because it includes the source filename and other settings) and the source?

**Comment by poiresel** [comment]
I deployed using `truffle deploy` which you're right uses just `solc --metadata` in the metadata section, i can confirm though that the metadata.json does match up with the build/contracts/*.json file metadata section once you jsonify the string.

I've attached the files that you've requested.
ERC20TokenFaucet.txt is what was used for the truffle deploy 
[ERC20TokenFaucet.txt](https://github.com/ethereum/source-verify/files/3858936/ERC20TokenFaucet.txt)
The metadata portion of the json file is the metadata output from truffle.

Source is available here:
https://ipfs.infura.io/ipfs/Qmbu6TMyKCH9HJVvhG6BFA3UkrJuGVdZjhFQqKRRKmsNyD

**Comment by poiresel** [comment]
That's good to hear! Every time I've tried to upload a file today, I kept receiving that 'Error: Metadata file not found. Did you include "metadata.json"?' Maybe I'll see if the programmatic post works instead.

Regarding, support truffle deploy json object, that would be helpful since then we can build a truffle plugin so that it's an easy script for truffle users though I'd prefer that we use --metadata-literal so the deployer's path to a contract is not uploaded.
